instrumentation/jdbc/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jdbc/PreparedStatementInstrumentation.java

```diff
@@ -53,6 +55,18 @@ public void transform(TypeTransformer transformer) {
     transformer.applyAdviceToMethod(
         named("addBatch").and(takesNoArguments()).and(isPublic()),
         PreparedStatementInstrumentation.class.getName() + "$AddBatchAdvice");
+    transformer.applyAdviceToMethod(
+        nameStartsWith("set")
+            .and(takesArguments(2))
+            .and(takesArgument(0, int.class))
+            .and(isPublic()),
+        PreparedStatementInstrumentation.class.getName() + "$Set2Advice");
+    transformer.applyAdviceToMethod(
+        nameStartsWith("set")
+            .and(takesArguments(3))
+            .and(takesArgument(0, int.class))
+            .and(isPublic()),
+        PreparedStatementInstrumentation.class.getName() + "$Set3Advice");
```

Comment:
I think it might be better to be a bit selective about these parameters. For example you probably can't do anything reasonable with the value passed to `setBinaryStream`. I'd choose the ones that you can convert to string reasonably and ignore the rest.